internal/storage/storage.go

```diff
@@ -27,8 +34,21 @@ type StorageWithCallbacks interface {
 }
 
 type storage struct {
-	data      map[string]any
-	callbacks map[string]StorageCallbackFunc
+	data        map[string]any
+	callbacks   map[string]StorageCallbackFunc
+	jsonStorage *configuration.JsonStorage
+}
+
+func (s *storage) Refresh(config configuration.Configuration, key string) error {
+	return s.jsonStorage.Refresh(config, key)
```

Comment:
so we will rely on GAF's jsonStorage impl? 
I think that makes sense.

I'm wondering though how it will be in the CLI. 
CLI will use JsonStorage's impl in this case that makes me wonder if we need the interface really since all impl are the same or is there a difference?